### PR TITLE
Formalize swarm process

### PR DESCRIPTION
**Pull request by oskarth**
status: ready

### CODE REVIEW

File: ideas/1XX-nail-swarm-process.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+evaluator? Possibly UX for communicating things during Town Hall, etc.
+
+### Requirements & Dependencies
+
+### Minimum Viable Product
+Goal Date: 2017-04-06
+Description:
+i. Ensure new draft / in-progress ideas use PR review process.
+ii. Measure Core/Swarm OKR coverage and communicate this/needs.
+iii. Evaluation terminology and expectations are clarified in manual.
```
Comment (oskarth) [inline ideas/1XX-nail-swarm-process.md]: Right now there's conflation on several dimensions:
- external (reality) vs internal self/group-evaluation
- as a person/role or as a mechanism
- as acting on behalf of something or someone (like users)

This has been talked about a lot in various contexts, but there's a confusion and lack of clarity around terminology, as well as difference between structural rules and customs (common law style).

Comment (oskarth) [reply ideas/1XX-nail-swarm-process.md]: For type 1 swarms, this ties into discussion around proportional payouts (internal self-evaluation) and anon evaluation panels (external interface).

File: ideas/1XX-nail-swarm-process.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+cultivated. One is that Status the company wants to ship a product, and the
+other is creating a decentralized autonomous organization. Since we are moving
+towards the latter, the structures we are building must be compatible with this
+latter vision, while still being mindful of the first.
+
+The primary difference between the two is that in the former, we are speaking on
+behalf of users, and people are generally compensated on a salary or salary-like
+basis.
+
+We thus distinguish between 'swarms' (type 1) more generally and 'Status LLC
```
Comment (martinklepsch) [inline ideas/1XX-nail-swarm-process.md]: Would suggest to come up with a better label for `swarms more generally` here. Maybe "swarms using fully open processes" or something. We have a problem where people are not necessarily aware of what the purpose of swarms is so I think being extra clear could help with that.

**Comment by martinklepsch** [comment]
Happy to contribute to this as Evaluator / Contributor / Advisor (5-10hr). 

Also committing to pushing a related idea to compensation experiments as outline in Success Metric 3 by Monday `2018-03-30`.

**Comment by oskarth** [comment]
@martinklepsch addressed comments, ready to merge

File: ideas/120-swarm-process.md [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+towards the latter, the structures we are building must be compatible with this
+latter vision, while still being mindful of the first.
+
+The primary difference between the two is that in the former, we are speaking on
+behalf of users, and people are generally compensated on a salary or salary-like
+basis.
+
+We thus distinguish between swarms (Type-1) as a general concept for bringing
+ideas to fruition and getting compensated by it, and 'Status LLC using swarms'
+(Type-2). The fatter might have more constraints given its immediate goals
```
Comment (vkjr) [inline ideas/120-swarm-process.md]: @oskarth, did you mean "later" or "fatter" is a correct word? :)

Comment (oskarth) [reply ideas/120-swarm-process.md]: Typo indeed!

Comment (oskarth) [reply ideas/120-swarm-process.md]: fixed https://github.com/status-im/ideas/blob/master/ideas/120-swarm-process.md

**Comment by oskarth** [comment]
Also fixed some fmt https://github.com/status-im/ideas/blob/master/ideas/120-swarm-process.md, think there was additional comment somewhere but don't see it anymore

**Pull request merged**